05.ls/ls01.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+def main
+  row_num = 3
+  all_files = get_all_files(row_num)
+  column_num = all_files.length / row_num
+  transposed_all_files = get_transposed_all_files(row_num, all_files, column_num)
+  files_to_display(transposed_all_files)
+end
+
+def get_all_files(row_num)
+  files = Dir.glob('*').sort
+  new_files = files.push(' ') while files.length % row_num != 0
+  new_files
```

Comment:
`Array#push`は破壊的なメソッドなので、新しく変数に入れる意味もなさそうですね。

```suggestion
  files = Dir.glob('*').sort
  files.push(' ') while files.length % row_num != 0
  files
```

こうで良いと思います。
もうちょっというと、このスペース詰めはのちに`transpose`するための前処理だと思うので、`get_transposed_all_files`の一行目でやった方が良いのではないかと思いました。`get_all_files`というメソッドなのに`transpose`のための前処理もやっているというのはちょっと違和感があります。ファイル名を取ってくるだけなのに、何で末尾にスペース入ってるんだろ？という感じですね。これも単一責任の原則に若干反している気がします。